sauce/features/accounts/toggle-splits/index.js

```diff
@@ -0,0 +1,85 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
+
+function isNotSubTransaction(transaction) {
+  const displayItemType = transaction.get('displayItemType');
+  return displayItemType !== ynab.constants.TransactionDisplayItemType.SubTransaction &&
+          displayItemType !== ynab.constants.TransactionDisplayItemType.ScheduledSubTransaction;
+}
+
+export class ToggleSplits extends Feature {
+  get idName() {
+    return 'toggle-splits';
+  }
+
+  constructor() {
+    super();
+    this.expanded = false;
+  }
```

Comment:
This is just a style thing, but it'd be great to use member initialisation instead of the constructor just in case we change the signature we need. So it'd look like this and you wouldn't need a constructor at all:

```javascript
export class ToggleSplits extends Feature {
  expanded = false

  get idName() {
    return 'toggle-splits';
  }

// ...etc...
}
```